internal/rpc/rpcserver/server.go

```diff
@@ -1822,6 +1822,8 @@ func (s *walletServer) RevokeTicket(ctx context.Context, req *pb.RevokeTicketReq
 		if err != nil {
 			return nil, status.Errorf(codes.InvalidArgument, "%v", err)
 		}
+	} else {
+		return nil, status.Errorf(codes.InvalidArgument, "%v", err)
```

Comment:
`err` is always going to be `nil` in this case, so it's maybe better to return a literal `TicketHash is a required argument` or something to that effect.